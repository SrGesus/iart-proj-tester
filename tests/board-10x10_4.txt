VE	LH	VC	FB	VB	VB	FB	VD	VD	VD
BC	FE	FE	LH	LH	VE	VC	LH	LH	LV
LH	FC	BD	VD	LV	FD	VE	BB	VB	LV
BE	FD	VD	VE	FB	VB	LV	LH	LV	BC
LH	FE	BE	BE	FC	VB	VE	BD	VB	LH
LH	FE	BE	FE	FE	LH	LV	FC	VC	BC
LV	FB	BE	BD	BC	BE	BE	FC	FC	BC
LH	FC	FB	BE	VB	FC	FE	LH	LV	BD
VD	BC	LV	BC	BE	LV	FB	VB	VC	LH
FD	LV	LV	BC	BD	FC	FE	VB	VC	VD
