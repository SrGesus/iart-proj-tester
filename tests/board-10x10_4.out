VB	LH	VE	FB	VB	VE	FD	VE	VB	VE
BD	FE	FC	LV	LV	VD	VE	LV	LV	LV
LV	FD	BB	VC	LV	FB	VD	BC	VC	LV
BD	FE	VD	VE	FC	VD	LH	LH	LH	BE
LV	FD	BB	BE	FD	VE	VB	BB	VE	LV
LV	FD	BE	FC	FB	LV	LV	FC	VD	BE
LV	FD	BC	BB	BC	BC	BC	FE	FD	BE
LV	FB	FD	BE	VB	FE	FD	LH	LH	BE
VD	BC	LH	BE	BD	LH	FE	VB	VE	LV
FD	LH	LH	BC	BC	FE	FD	VC	VD	VC
